particles/Loadmesh.cpp

```diff
@@ -8,7 +8,7 @@ meshdata attrib_to_data(tinyobj::ObjReader reader, std::string inputfile, tinyob
       if (!reader.Error().empty()) {
           std::cerr << "TinyObjReader: " << reader.Error();
       }
-      exit(1);
+      abort();
```

Comment:
It's better to use abort because this way the debugger will stop whenever we reach that instead of just exiting.